circle.yml

```diff
@@ -2,6 +2,9 @@ machine:
   python:
     version: 2.7.6
 
+dependencies:
+  override:
+    - pip install tox
```

Comment:
I was getting some whacky behavior where it wasn't respecting my tox.ini until I specifically overrode the dependency behavior.